config/mailer.yml

```diff
@@ -0,0 +1,67 @@
+languages:
+  - code: EN
+    name: English
+  - code: RU
+    name: Russian
+
+keychain:
+  barong:
+    algorithm: RS256
+    value: "LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUF4RHVTTGVQbnBiS3JYTkhQayt6Vwo0b29LQjk1SnlQVk5VaGNEOHJJNDIyL1dpYk5EOUNydWxwOERKYzRRUWdVWnpwVGJBWUxNekxwUHJGUjBkSFpiCjZhQ3dSemlRR2I1WHVRcnZucGZHZG5YNkRlZHdNK2hwN0NlS1ZDaXZqcHhITzF3SEpxTkQwRmRTbUlKTEJhcnkKclpJWitHckZhdE50eDdRQllURnJrRUVwVmdNZkRWWnYyWnNNdmYwNzcvQkE1UlZXd0ZwakZFRHZYaitQcTJDbgpNWUo0QkZMMEpFS0NPbWVMblBVVGl0R1BKdDdxdGw0UmN3WUthb3FxS2gvNDg4ZnBJakx3QmIzZFg1YW5GRzFLCk1rUTdqNXEwem1aeHYrWDdKaGpNMjhpUyt5WUo2TzJQcGdWdGhvc1QrSnZZL1pPeHZiQjJmRkZsdnRRaUdLYzEKSVFJREFRQUIKLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0tCg=="
+  peatio:
+    algorithm: RS256
+    value: "changeme"
+
+exchanges:
+  barong_system:
+    name: barong.events.system
+    signer: barong
+  barong_model:
+    name: barong.events.model
+    signer: barong
+  peatio:
+    name: peatio.events.model
+    signer: peatio
+
+events:
```

Comment:
I think this config should be mail only
also we can name this part of the code postmaster
the service name and it's config

Reply:
Renamed to postmaster.